main.js

```diff
@@ -0,0 +1,197 @@
+    rebuiltArray = [boardSubArray1,boardSubArray2,boardSubArray3]
+    return rebuiltArray
+  }
+
+//checks wins vertically and horizontally
+checkWinUpDownAndAcross = (someArray) => {
+  for(var i = 0; i<3;i++) {
+    var colSum = "";
+    for(var j = 0; j<3;j++){
+        colSum += someArray[j][i];
+    }
+    if (colSum === ("OOO")) {
+    cancelEventListeners()
+    statusArea.textContent = "Circle's Win"
+    return true
+  } else if (colSum === "XXX") {
+    cancelEventListeners()
+    statusArea.textContent = "Cross's Win";
+    return true
+    }
+  }
+    
+  for(var i = 0; i<3;i++){
+      var rowSum = "";
+      for(var j = 0; j<3;j++){
+          rowSum += someArray[i][j];
+      }
+      if (rowSum === ("OOO")) {
+      cancelEventListeners()
+      statusArea.textContent = "Circle's Win"
+      return true
+      } else if (rowSum === "XXX") {
+        cancelEventListeners()
+        statusArea.textContent = "Cross's Win";
+        return true
+      }
+    }
+  }
+
+//checks diagnol wins
+checkWinDiagnol = (someArray) => {
+  if (someArray[0]+someArray[4]+someArray[8] === "OOO"){
+      cancelEventListeners()
+      statusArea.textContent = "Circle's Win"
+      return true
+}
+  else if (someArray[0]+someArray[4]+someArray[8] === "XXX"){
+      cancelEventListeners()
+      statusArea.textContent = "Cross's Win"
+      return true
+  }
+  else if (someArray[2]+someArray[4]+someArray[6] === "OOO"){
+    cancelEventListeners()
+    statusArea.textContent = "Circle's Win"
+    return true
+}
+  else if (someArray[2]+someArray[4]+someArray[6] === "XXX"){
+    cancelEventListeners()
+    statusArea.textContent = "Cross's Win"
+    return true
+  }
+}
+//check stalemate not working.... yet
+/* checkStalemate = (count,checkwin1,checkwin2) => {
+  if (count === 9) {
+    if (checkwin1 || checkwin2) {
+      statusArea.textContent = "Stalemate"
+    }
+  }
+} */
+
+reset = () => {
+  window.location.reload(true);
+}
+
+
+// game clock function
+ gameClock = (playing=true) => {
+  if (playing) {
+  let startTime = Date.now()
+  time = setInterval(function () {
+      let now = new Date().getTime()
+      let timer = now - startTime
+      let days = Math.floor(timer / (1000 * 60 * 60 * 24));
+      let hours = Math.floor((timer % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+      let minutes = Math.floor((timer % (1000 * 60 * 60)) / (1000 * 60));
+      let seconds = Math.floor((timer % (1000 * 60)) / 1000)
+      document.getElementById("timer").innerHTML = days + "d " + hours + "h "
+      + minutes + "m " + seconds + "s "
+  })} else {
+      clearInterval(time)
+        }
+  }
+  
+
+
+
+//creates global game object
+let game = new Game();
+
+//starts game
+  let startGame = () => {
+    startButton.disabled = true;
+    let whosTurn = game.playerTurn();
+    game.printMessages(whosTurn);
+    setUpEventListeners()
+    gameClock()
+  };
+
+let count = 0
+// plays game
+ function play (evt) {
```

Comment:
It's generally considered better organization to keep all of your function definitions together.